deployments.md

```diff
@@ -2,13 +2,19 @@
 
 *You now need yarn installed - `npm install -g yarn`*
 
- 1. Update the compiler submodule pointer to the tagged release.
+ 1. Update the package version number in `package.json` at
+    https://github.com/chadkillingsworth/closure-compiler-graal.
+    The major version should be equal to the compiler version and the minor and patch numbers should be `0`.
+    Example: 20160619.0.0.
+    Wait for Travis to publish the new versions.
```

Comment:
I'm unclear on what "Wait for Travis to publish the new version."

Is there any problem with using `npm version patch` or `npm version minor`.

Does the major version have to be today's date?

Reply:
If you bump the version number in the package.json on that repo (and this one), Travis will build and automatically publish the new versions to npm.

So far, I'm the only person that has published minor or patch versions. They work just great. The instruction were written for the rest of the compiler team. As each compiler monthly release is a new major version, the instructions were written to make sure that the major version increment was performed correctly.

The major version is always equal to the compiler version being released. Since the compiler is date versioned, we decided a long time ago to make that the major version of the package. The date is almost never today's date. Usually when a build is tagged it's a couple of days later before it's pushed to external repos like maven and npm.